### PR TITLE
Introduce SourceLair Console

### PR DESCRIPTION
**Pull request by TDaglis**

### CODE REVIEW

File: _bootcamp/console.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: page
+---
+
+SourceLair Console is used to log specific user actions, along with their success state. Currently, the actions logged in SourceLair Console are:
+
+- Django - Start app
+- Source Control - Checkout a branch
+- Source Control - Create new branch
+
+The Console is located in the bottom pane of SourceLair:
```
Comment (akalipetis) [inline _bootcamp/console.md]: Could we name this "Terminals sectio", or "Console section"?

Comment (akalipetis) [inline _bootcamp/console.md]: I'm talking about "bottom pane"

**Comment by akalipetis** [comment]
LGTM :bear: just a minor proposal.

File: _bootcamp/console.md (hunk 1 above)
Comment (TDaglis) [inline _bootcamp/console.md]: That's the way I originally had it. But it looks weird. The console is in the console

Comment (akalipetis) [inline _bootcamp/console.md]: :+1:

**Pull request merged**